src/modules/launcher/Plugins/Microsoft.Plugin.Calculator/CalculateEngine.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using Mages.Core;
+
+namespace Microsoft.Plugin.Calculator
+{
+    public class CalculateEngine
+    {
+        private readonly Engine _magesEngine = new Engine();
+
+        public CalculateResult Interpret(string input)
+        {
+            return Interpret(input, CultureInfo.CurrentCulture);
+        }
+
+        public CalculateResult Interpret(string input, CultureInfo cultureInfo)
+        {
+            if (input == null)
```

Comment:
Can you please do a `string.IsNullOrWhiteSpace` check ?

Reply:
Also, this same check is here happening here and in `CalculateHelper.InputValid` as well. So, you might want to remove it from here and let it be handled by `CalculateHelper.InputValid` function.

Reply:
I have removed the check, so the only place that it checks is CalculateHelper.